tests/e2e-pw/tests/customer-account/customer-account.block_theme.spec.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import { expect, test } from '@woocommerce/e2e-playwright-utils';
+import { getBlockByName } from '@woocommerce/e2e-utils';
+
+const blockData = {
+	name: 'woocommerce/customer-account',
+	selectors: {
+		frontend: {
+			icon: 'svg',
+			label: '.label',
+		},
+		editor: {
+			iconOptions: '.customer-account-display-style select',
+			iconToggle: '.wc-block-customer-account__icon-style-toggle',
+		},
+	},
+};
+
+const publishAndVisitPost = async ( { page, editor } ) => {
+	await editor.publishPost();
+	await page.waitForLoadState( 'networkidle' );
+	const url = new URL( page.url() );
+	const postId = url.searchParams.get( 'post' );
+	await page.goto( `/?p=${ postId }`, { waitUntil: 'networkidle' } );
+};
+
+const selectTextOnlyOption = async ( { page } ) => {
+	await page
+		.locator( blockData.selectors.editor.iconOptions )
+		.selectOption( 'Text-only' );
+
+	await expect(
+		await page.locator( blockData.selectors.editor.iconToggle )
+	).not.toBeVisible();
+};
+
+const selectIconOnlyOption = async ( { page } ) => {
+	await page
+		.locator( blockData.selectors.editor.iconOptions )
+		.selectOption( 'Icon-only' );
+
+	await expect(
+		await page.locator( blockData.selectors.editor.iconToggle )
+	).toBeVisible();
```

Comment:
nit: is `await.expect` needed here? How about:
```suggestion
	await page.locator( blockData.selectors.editor.iconToggle ).toBeVisible();
```